packages/lib-core/src/testing/mocks.ts

```diff
@@ -0,0 +1,31 @@
+import type { AnyObject } from '@monorepo/common';
+import { noop } from 'lodash';
+import type { PluginManifest } from '../types/plugin';
+import type { PluginEntryModule } from '../types/runtime';
+
+export const mockPluginManifest = ({
```

Comment:
Are these 2 utilities something we really want to expose/maintain? I'd imagine most apps/plugins consuming the SDK and writing tests will be providing their own test manifest/entry module.

Reply:
That's a good question.

Mocking a plugin manifest (like in `mockPluginManifest`) is very simple so this could be offloaded to consumers.

Mocking an entry module (like in `mockPluginEntryModule`) is a bit more complex, but still most typically a simple transformation from static `pluginModules: { [moduleRequest: string]: AnyObject }` declaration to SDK expected `get` function `(moduleRequest: string) => Promise<() => AnyObject>` - I don't think it makes sense to test asynchronicity within that function, so the resulting Promise would most typically be `Promise.resolve(() => pluginModules[moduleRequest]` or similar.

Overall, I think you're right here. We don't have to expose these functions.